agent_go/update_eventbridge.sh

```
@@ -0,0 +1,4 @@
+#!/bin/bash
+for file in $(find pkg/orchestrator/agents -name "*.go"); do
```

Shellcheck (0.11.0)

[warning] For loops over find output are fragile. Use find -exec or a while read loop.

(SC2044)